src/microbe_stage/MicrobeHUD.cs

```diff
@@ -395,8 +398,10 @@ protected override void UpdateAbilitiesHotBar()
 
         bool showToxin;
         bool showSlime;
+        bool showMucocyst = false;
```

Comment:
```suggestion
        bool showMucocyst;
```
I think automatic checks should have complained about this...

Reply:
Then the warning caught a bug, you aren't updating the mucocyst state for player who is in a colony.